From c2d7dfc68ffcb389c9f73b5ef94da7b270bdcf9e Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Sat, 22 Jun 2019 19:38:48 -0600
Subject: [PATCH] Add missing cairo-svg dependency to test-ot-color

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
c2d7dfc68ffcb389c9f73b5ef94da7b270bdcf9e]
---
 src/test-ot-color.cc | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/test-ot-color.cc b/src/test-ot-color.cc
index 44bd2eb6..e9126c06 100644
--- a/src/test-ot-color.cc
+++ b/src/test-ot-color.cc
@@ -25,7 +25,9 @@
 
 #include "hb.hh"
 
-#ifndef HB_NO_COLOR
+#include <cairo.h>
+
+#if !defined(HB_NO_COLOR) && defined(CAIRO_HAS_SVG_SURFACE)
 
 #include "hb-ot.h"
 
@@ -35,7 +37,6 @@
 #include FT_FREETYPE_H
 #include FT_GLYPH_H
 
-#include <cairo.h>
 #include <cairo-ft.h>
 #include <cairo-svg.h>
 
-- 
2.20.1

